constructors/bq/test/package-lock.json

```diff
@@ -1,1036 +1,1117 @@
 {
-    "name": "constructors_bq_integration_tests",
```

Comment:
IMO I think this file can be added to `.gitignore`

Reply:
yes, I just added it and seems to pass all the tests.